src/common/AndroidUtils.ts

```diff
@@ -589,18 +588,44 @@ export class AndroidSDKUtils {
         return `'${pathName}'`;
     }
 
-    private static enableHWKeyboardForEmulator(
+    private static updateEmulatorConfig(
```

Comment:
Nit: I like to decorate Promise-based methods with `async`, just so you can `await` them if/when the need arises.